README.md

```diff
@@ -209,18 +209,24 @@ There are 4 files you must provide:
    resource][chrMap] provides such files for various organisms, and in the
    expected format.
 
+5. **OPTIONAL**: A **BED6** file with regions for which to produce
+   [ASCII-style alignment pileups][ascii-pileups] for. If not provided, no
+   pileups will be generated. Refer [here][bed-format] to see the expected
+   format.
```

Comment:
* One `for` is enough here: _"for which to produce [...] ~for~"_
* Better: `See here for the expected format` rather than `Refer here to see the expected format`